service/src/main/java/com/commercetools/util/SphereClientConfigurationUtil.java

```diff
@@ -1,99 +1,55 @@
 package com.commercetools.util;
 
 import io.sphere.sdk.client.BlockingSphereClient;
-import io.sphere.sdk.client.QueueSphereClientDecorator;
-import io.sphere.sdk.client.RetrySphereClientDecorator;
 import io.sphere.sdk.client.SphereClient;
 import io.sphere.sdk.client.SphereClientConfig;
-import io.sphere.sdk.client.SphereClientFactory;
-import io.sphere.sdk.retry.RetryAction;
-import io.sphere.sdk.retry.RetryPredicate;
-import io.sphere.sdk.retry.RetryRule;
+import io.sphere.sdk.client.retry.RetryableSphereClientBuilder;
+import io.sphere.sdk.http.AsyncHttpClientAdapter;
+import io.sphere.sdk.http.HttpClient;
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig;
 
 import javax.annotation.Nonnull;
-import java.time.Duration;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Random;
 import java.util.concurrent.TimeUnit;
 
-import static io.sphere.sdk.http.HttpStatusCode.BAD_GATEWAY_502;
-import static io.sphere.sdk.http.HttpStatusCode.GATEWAY_TIMEOUT_504;
-import static io.sphere.sdk.http.HttpStatusCode.SERVICE_UNAVAILABLE_503;
-
 public final class SphereClientConfigurationUtil {
     private static final long DEFAULT_TIMEOUT = 10;
     private static final TimeUnit DEFAULT_TIMEOUT_TIME_UNIT = TimeUnit.SECONDS;
-    protected static final int RETRIES_LIMIT = 5;
-    private static final int MAX_PARALLEL_REQUESTS = 30;
```

Comment:
@leungkinghin Default value for maximum parallel requests is now 20; do you think we should increase to 30 or use the defaults?

Reply:
I would prefer to keep it as 30, it is default value used in payone-integration before the change.

Reply:
4e6901d